blacklistfilter/evaluator/evaluator.py

```diff
@@ -0,0 +1,316 @@
+import csv
+import json
+from queue import Queue
+import os
+import sys
+import signal
+import time
+from threading import Thread, Event
+import requests
+from datetime import datetime
+import pytrap
+import shutil
+from optparse import OptionParser
+from collections import defaultdict
+
+#Enter an API key to obtain informations from nerd
+NERD_API_KEY = None
+
+#Constant defined in Adaptive Filter
+ADAPTIVE_BLACKLIST_ID = 999
+
+parser = OptionParser(add_help_option=True)
+parser.add_option("-i", "--ifcspec", dest="ifcspec",
+                  help="TRAP IFC specifier", metavar="IFCSPEC")
+parser.add_option("-c", "--csv-path", dest="csv", help="Path to the csv files generated by Split Evidence")
+parser.add_option("-e","--evidence-path", dest="evidence", help="Path to the folder for storing data of unreported detections")
```

Comment:
If the config file is passed using a new option, you need not to use configure to fill the correct path as it is in `blacklist_downloader`.